tools/block-generator/generator/server.go

```diff
@@ -60,17 +62,19 @@ func MakeServerWithMiddleware(configFile string, addr string, blocksMiddleware B
 	gen, err := MakeGenerator(config)
 	util.MaybeFail(err, "Failed to make generator with config file '%s'", configFile)
 
-	mux := http.NewServeMux()
-	mux.HandleFunc("/", help)
-	mux.Handle("/v2/blocks/", blocksMiddleware(http.HandlerFunc(getBlockHandler(gen))))
-	mux.HandleFunc("/v2/accounts/", getAccountHandler(gen))
-	mux.HandleFunc("/genesis", getGenesisHandler(gen))
-	mux.HandleFunc("/report", getReportHandler(gen))
-	mux.HandleFunc("/v2/status/wait-for-block-after/", getStatusWaitHandler(gen))
+	r := mux.NewRouter()
+	r.HandleFunc("/", help)
+	r.Handle("/v2/blocks/{round}", blocksMiddleware(http.HandlerFunc(getBlockHandler(gen))))
+	r.HandleFunc("/v2/accounts/", getAccountHandler(gen))
+	r.HandleFunc("/genesis", getGenesisHandler(gen))
+	r.HandleFunc("/report", getReportHandler(gen))
+	r.HandleFunc("/v2/status/wait-for-block-after/", getStatusWaitHandler(gen))
+	r.HandleFunc("/v2/ledger/sync/", func(w http.ResponseWriter, r *http.Request) {})
```

Comment:
TODO?

Reply:
it's a noop for this endpoint because we don't need to "pause" the generator.

Reply:
the overflow error is triggered at this line, https://github.com/algorand/go-algorand/blob/master/ledger/store/trackerdb/sqlitedriver/schema.go#L220

Reply:
Looking at the logic, it must be happening in [AddAccount()](https://github.com/algorand/go-algorand/blob/f4f5ec6d058262eba330ba2c4674032e4286c691/ledger/ledgercore/totals.go#L82) and in Algos-only code. This overflow would never happen (for legit transactions) if we used a realistic number of Algos. And for assets, the overflow wouldn't occur for legit transactions for similar reasons.

Resolving.